packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3605,8 +3605,8 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   closeRemoteStreams() {
-    const {remoteAudioStream, remoteVideoStream, remoteShareStream, shareAudioStream} =
-      this.mediaProperties;
```

Comment:
Why are we doing this PR on next and not beta branch? Beta is the only source of truth for plugin meetings, the agreement was that for plugin-meetings we will take all of the source from beta. There should be nothing on the next branch apart from maybe some package.json changes

Reply:
The problem here is that `shareAudioStream` isn't a remote stream and it is of type `LocalSystemAudioStream`

This is causing issue when we make call to `stopStream` which only accepts `RemoteStream.` Wasn't going to make this a permanent change. Already had a chat with Bryce about this yesterday and he wanted me to discuss this with you. That's why added a Todo

Reply:
@mkesavan13 how do we reconcile this with the code that's in `beta`?

Reply:
This is a real bug, it exists on beta too, so it should be fixed on beta. It was probably introduced when the WCME 3.0 changes were done and has gone unnoticed, because closeRemoteStreams() is only called on meeting leave and if it accidentally stops local share audio stream, it's not causing any real problem in the web app, because we always have share stopped anyway when we leave the meeting